src/foam/nanos/auth/email/EmailVerificationCode.js

```diff
@@ -178,10 +178,10 @@ foam.CLASS({
     },
     {
       name: 'resendCode',
-      isAvailable: function(showAction) {
-        return showAction;
+      isAvailable: function(codeVerified) {
```

Comment:
can we also include showaction here? when the resend code is a dynamicaction on wizardlet, we don't want to show the action in the detailview even if code has not been verified